projects/ccf-eui/src/app/app.component.ts

```diff
@@ -90,6 +93,7 @@ export class AppComponent {
     right.open();
     right.closeExpanded();
     filterbox.removeBox();
+    this.bodyUI.setupBodyUI();
```

Comment:
setupBodyUI is an _awful_ big hammer to use. It creates a new instance of deck.gl each time. Why not try `zoomToBounds`? to zoom out to the body? or something closer to that. If you absolutely can't get it this way, at least use `recreateBodyUI` so that it cleans up after itself.